org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/ContentModelCompletionParticipant.java

```diff
@@ -352,15 +372,29 @@ private void fillAttributeValuesWithCMAttributeDeclarations(CMElementDeclaration
 				item.setKind(CompletionItemKind.Value);
 				item.setFilterText(insertText);
 				item.setTextEdit(Either.forLeft(new TextEdit(fullRange, insertText)));
-				MarkupContent documentation = XMLGenerator.createMarkupContent(cmAttribute, value, cmElement, request);
-				item.setDocumentation(documentation);
+
+				if (request.isResolveDocumentationSupported()) {

```

Comment:
I think texedit should be done in resolve process to minimize the json response size when thére are a of of items to return

Reply:
I could look into adding the text edits to `additionalTextEdit` and send a blank `textEdit` for the unresolved completion item, but I don't know if its worthwhile, and the documentation seems to discourage this.

Reply:
Thanks for your investigation. Let's keep like you did with textEdit fill.